src/style.css

```diff
@@ -1,36 +1,39 @@
-.navigation .nav-header {
-  padding: 0;
-  margin: 0;
-  list-style: none;
-  display: flex;
-  gap: 1rem;
-}
-
 .navigation {
-  color: white;
+  padding: 24px 4%;
+  background-color: #fff;
+  border-bottom: 1px solid #e8e8e8;
   display: flex;
-  justify-content: space-between;
-  align-items: stretch;
-  gap: 2rem;
-  padding: 0 1rem;
+  grid-gap: 40px;
+  gap: 40px;
 }
```

Comment:
- [ ] The navigation panel covers the whole width and I would suggest you decrease the height of the panel
- [ ] I think it would be better if you decrease the size of the logo and place that logo on the right side as shown in the design
### Your design
![his](https://user-images.githubusercontent.com/81727988/181935056-cf929471-27c0-4d9b-8ca2-9d9d8bd6b258.png)
### Figma design
![figma](https://user-images.githubusercontent.com/81727988/181935067-39ed543a-3dac-403b-bc5a-fa3ac00e5f1d.png)